core/common/src/Base64EncodedString.ts

```diff
@@ -0,0 +1,65 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module Entities
+ */
+
+/** Represents an array of bytes encoded in base-64 with a prefix indicating the encoding, as persisted in an [ECDb]($backend) for properties of `binary` type.
```

Comment:
i'm not sure this sentence is right. The property is persisted as binary in the ECDb, but retuned using Base64EncodedString.

Reply:
Yeah I'll rephrase.

Reply:
Rephrased.